collision-handlers.rb

```diff
@@ -0,0 +1,58 @@
+require 'gosu'
+require 'chipmunk'
+require_relative 'player'
+require_relative 'obj'
+
+class PlayerPlatformCollisionHandler
+  def initialize(player)
+    @player = player
+  end
+
+  def begin(a, b, arbiter)
+
+    if arbiter.normal(0).y < 0
+      arbiter.ignore
+      false
+    else
+      @player.j = true
+    end
+    true
```

Comment:
Because if the if statement is true then it will stop collision and if it is false then i need to return true for the collision to continue.